### ISSUE TITLE
Return None if no rows are found in helper functions

### ISSUE THREAD

**Issue by AliMickey** [pgjones/quart-db#20]
I am attempting to use fetch_one and fetch_all to return a user based on a provided email. 

If the email does not exist, it will throw an exception instead of returning None due to:

```
File "D:\Programming\Git\aikido-suite\.env\Lib\site-packages\quart_db\backends\aiosqlite.py", line 124, in fetch_one
    return {key: row[key] for key in row.keys()}
                                     ^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'keys'
```

I believe it is beneficial to have a check for row count and return a None instead for my function to handle.

**Comment by pgjones** [comment]
Thanks fixed in 83d069bd73be02bb97fd42134e591db44f9b2751

**Comment by pgjones** [comment]
Released in 0.8.0
